src/components/dialog/NewProjectTemplateDialog.vue

```diff
@@ -21,7 +21,6 @@
           navigation
           padding
           arrows
-          height="200px"
```

Comment:
keep this, because if svg don't have default it will not work

Reply:
replaced with `min-height="200px"` to be able to enlarge the carousel image